test/rules/member-ordering/method/test.ts.lint

```diff
@@ -19,4 +19,18 @@ class Foo {
     ~~~~~~~~~~ [0]
 }
 
+interface IArrowMethods {
+    a: number;
+    b(): void;
+    c: () => void;
+}
+
+class ArrowMethods {
+    public a = 5;
+    public b = "foo";
+    public c() {}
+    public d = () => {};
+    public e = function() {};
```

Comment:
I don't think this rule checks anything about alphabetization. Second point seems like a good idea though